legacy_ibc_testing/core/events.go

```diff
@@ -36,7 +36,7 @@ func ReconstructPacketFromEvent(event abci.Event) (packet types.Packet, err erro
 		return packet, err
 	}
 	return types.NewPacket(
-		attrMap[string(types.AttributeKeyData)], // data
+		attrMap[string(types.AttributeKeyDataHex)], // data
```

Comment:
What does this change do? Doesn't seem like a lint

Reply:
old call is deprecated and I think that these both return the same thing